main/dhcp/APKBUILD

```diff
@@ -67,13 +72,15 @@ package() {
 
 dhclient() {
 	pkgdesc="ISC dhcp client"
+	depends="bash"
 	install -d -o dhcp -g dhcp "$subpkgdir"/var/lib/dhcp
 	install -d -o dhcp -g dhcp "$subpkgdir"/var/run/dhcp
 	install -d "$subpkgdir"/usr/sbin
 	install -d "$subpkgdir"/sbin
 	install -d "$subpkgdir"/etc/dhcp
 	mv "$pkgdir"/usr/sbin/dhclient "$subpkgdir"/usr/sbin/
 	mv "$pkgdir"/sbin/dhclient-script "$subpkgdir"/sbin/
+	ln -s ../sbin/dhclient-script "$subpkgdir"/etc/dhclient-script
```

Comment:
should we no install the file in /etc/ instead of having a symlink? this will break generation of .apk-new files if the user has modified the script.

Also should the script not be in the /etc/dhcp directory?

Reply:
The setup here is the `dhclient` binary living in /sbin, it has a configuration file in `/etc/dhcp` and that configuration specifies where the `dhclient-script` script is located.

The long standing tradition for where that script lives is `/etc/dhclient-script` but for whatever reason it lives in `/sbin` on Alpine. If it was modified that's where the modified file would be.

Now these are the options I see:
- move the script from `/sbin` to `/etc/dhcp` in `pre-install`
- update the configured location for the script
- leave things as they were installed and let the user configure everything

I chose that last option. By adding a link from the traditional location to the installed script we prevent surprising failures when copying the example configuration or an old configuration into `/etc/dhcp` for example.

Reply:
it seems to live here `/sbin/dhclient-script` in Debian as well, so its probably no intended to be modified at all. Hence the symlink should not be needed at all then, if where not for the example :smille: in /etc/dhcp/dhclient.conf.example... 

Also in Alpine is seems to currently live in `/sbin` https://pkgs.alpinelinux.org/contents?branch=edge&name=dhclient&arch=x86_64&repo=main

So yes...you are right the symlink is probably a good idea. 

What do you think should the example be patched to point to `/sbin/dhclient-script` perhaps?

Reply:
My opinion is that patching the example makes little sense. I don't see a strong argument for why the script belongs in `/sbin` as opposed to say `/usr/bin` but I don't care about this enough to change it.

I know my current configuration uses the `/etc/dhclient-script` location and so have most of the configurations I've seen others use.

Reply:
that it is in `/sbin` and not `/usr/sbin` at least in Debian is I guess historical for thin client setups where /usr was mounted via NFS later during boot.